cloudnet-modules/cloudnet-report/src/main/java/de/dytanic/cloudnet/ext/report/CloudNetReportModule.java

```diff
@@ -79,17 +81,28 @@ public void init() {
   public void initConfig() {
     this.getConfig().getBoolean("savingRecords", true);
     this.getConfig().getString("recordDestinationDirectory", "records");
+    this.getConfig().getBoolean("addCustomDate", true);
+    this.getConfig().getString("dateFormat", "yyyy-MM-dd");
     this.getConfig().get("pasteServerType", PasteServerType.class, PasteServerType.HASTE);
     this.getConfig().getString("pasteServerUrl", "https://just-paste.it");
     this.getConfig().getLong("serviceLifetimeLogPrint", 5000L);
-
     this.saveConfig();
+    if (this.getConfig().getBoolean("addCustomDate")) {
+      dateFormat = new SimpleDateFormat(this.getConfig().getString("dateFormat"));
+    }
   }
 
+
   @ModuleTask(order = 126, event = ModuleLifeCycle.STARTED)
   public void initSavingRecordsDirectory() {
```

Comment:
This should no longer be a module task, as we do init the directory inside the `CloudServicePreDeleteEvent`:
```suggestion
  public void initSavingRecordsDirectory() {
```